OracleJava/11/Dockerfile.8-slim

```diff
@@ -73,7 +72,7 @@ RUN set -eux; \
 # JDK assumes freetype is available		
 		freetype fontconfig \
 	; \
-	microdnf clean all; \
+	rm -rf /var/cache/yum; \
```

Comment:
This should be `/var/cache/dnf`

Reply:
Fixed.  Thanks

Reply:
I will use OL8 for JDK 17+ but will keep the current images on OL8 Slim to avoid surprising existing users.